governance-and-automation/automated-workflows/private-workspaces-shared-with-everyone-admin.md

```diff
@@ -0,0 +1,29 @@
+---
+description: This article explains how to edit the Private Workspaces Shared with Everyone policy. 
+---
+
+# Private Workspaces Shared with Everyone
+
+Syskit Point provides a predefined **Private Workspaces Shared with Everyone policy** that is applied tenant-wide.
```

Comment:
dok policy po defaultu je applyan tenant wide, treba malo razlučiti da ljudi shvate da nije ipak applyan na sve workspacese, pošto applya se na sve communication siteove, OneDriveove i samo Private teamove i grupe, Public grupe i teamovi nisu u djelokrugu tog policya.